### PR TITLE
Minimal gcc 4.9 Travis build coverage

### PR DESCRIPTION
**Pull request by nigels-com**
See also: https://travis-ci.org/nigels-com/tinyformat/builds/130141230

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - g++-4.9
+
+script:
+    - make CXX=g++-4.9 test
+
+addons:
+  apt:
+    sources:
+    - deadsnakes
+    - ubuntu-toolchain-r-test
```
Comment (c42f) [inline .travis.yml]: Do we need the `deadsnakes` ppa?  I assume the ubuntu-toolchain-r-test one is for g++?

Comment (nigels-com) [inline .travis.yml]: Oh, right!  deadsnakes is a python PPA, after all.

**Comment by nigels-com** [comment]
Deadsnakes PPA removed, good to go.

**Comment by c42f** [comment]
Great, thanks!

**Pull request merged**